js/main.js

```diff
@@ -75,14 +77,185 @@
     return pictureElement;
   };
 
+  // задание - личный проект: больше деталей в module3-task3
+  var bigPicture = document.querySelector('.big-picture');
+  var socialComment = bigPicture.querySelector('.social__comment');
+  var commentsCount = bigPicture.querySelector('.social__comment-count');
+  var commentsLoader = bigPicture.querySelector('.comments-loader');
+  var bigPictureCancel = bigPicture.querySelector('#picture-cancel');
+
+  var openBigPicture = function () {
+    bigPicture.classList.remove('hidden');
+  };
+
+  var closeBigPicture = function () {
+    bigPicture.classList.add('hidden');
+  };
+
+  var onBigPictureEscDown = function (evt) {
+    if (evt.keyCode === ESC_KEYCODE) {
+      bigPicture.classList.add('hidden');
+    }
+  };
+
+  var closeBigPicturEnterDown = function () {
```

Comment:
```suggestion
  var closeBigPictureEnterDown = function () {
```
орфографическая ошибка в слове picture